File: 1154.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var vezes = 0
var lista = 0
while(true){
    a = Number(lines.shift())
    if(a < 0){
        break;}
    var lista = lista + a
    var vezes = vezes + 1}
var c = lista / vezes
console.log(`${c.toFixed(2)}`)

File: 1154.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        Locale.setDefault(Locale.US);
        double vezes = 0, lista = 0;
        while(true) {
                double a = input.nextInt();
                if(a < 0) {
                        break;
                }
                lista += a;
                vezes += 1;
        }
        double c = (double)(lista / vezes);
        System.out.printf("%.2f%n",c);
}
}
